borg/archiver.py

```diff
@@ -466,8 +467,8 @@ def do_delete(self, args, repository):
                         msg.append(format_archive(archive_info))
                 msg.append("Type 'YES' if you understand this and want to continue: ")
                 msg = '\n'.join(msg)
-                if not yes(msg, false_msg="Aborting.", truish=('YES', ),
-                           env_var_override='BORG_DELETE_I_KNOW_WHAT_I_AM_DOING'):
+                if not yes(msg, false_msg="Aborting.", invalid_msg='Invalid answer, aborting.', truish=('YES', ),
```

Comment:
maybe we could use non-language defaults.
like "> " and "???" for invalid or so.
i don't want yes() to get into i18n department.